python/common/org/python/stdlib/math/math.java

```diff
@@ -0,0 +1,44 @@
+package python;
+
+@org.python.Module(
+        __doc__ =
+                "TODO"
+)
+public class math extends org.python.types.Module {
+    public math() {
+        super();
+    }
+
+    public static org.python.Object _STRUCT_TM_ITEMS;
+
+    @org.python.Attribute
+    public static org.python.Object __file__ = new org.python.types.Str("python/common/python/math.java");
+
+    @org.python.Method(
+         __doc__ = "Return the square root of number",
+        args ={"number"}
+    )
+    public static org.python.Object sqrt(org.python.Object number) {
+        if (number instanceof org.python.types.Float){
+            if(((org.python.types.Float) number).value < 0.0){
+                throw new org.python.exceptions.ValueError("math domain error");
+            }
+            return new org.python.types.Float(Math.sqrt(((org.python.types.Float) number.__float__()).value));
+        }
+        else if(number instanceof org.python.types.Int){
+            if(((org.python.types.Int) number).value < 0){
+                throw new org.python.exceptions.ValueError("math domain error");
+            }
+            return new org.python.types.Float(Math.sqrt(((org.python.types.Float) number.__float__()).value));
+        }
+        else if(number instanceof org.python.types.Bool) {
```

Comment:
In python booleans are instances of Int: therefore we will never enter this case since we already check for integers in an earlier if statement on row 28.